rdesktop.c

```diff
@@ -697,6 +700,16 @@ main(int argc, char *argv[])
 					p++;
 				}
 
+				if (*p == '@')
+				{
+					g_dpi = strtol(p + 1, &p, 10);
```

Comment:
If no DPI specified by argument, could we use DPI fetched from display ?

Would that be an good out of the box experience ?